src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -45,3 +49,33 @@ export function getCurlCommand(data) {
   result += '"';
   return result;
 }
+
+/**
+ * Converts an object of body/querystring key/value pairs
+ * into a tuple of [key, value] pairs, and sorts them.
+ *
+ * This handles the case for query strings that were decoded like so:
+ *
+ *   ?foo=bar&foo=baz => { foo: ['bar', 'baz'] }
+ *
+ * By converting them to [['foo', 'bar'], ['foo', 'baz']]
+ */
+export function objectToSortedTupleArray(obj) {
+  return Object.keys(obj)
+    .reduce((out, k) => {
+      let val = obj[k];
+      return out.concat(
+        {}.toString.call(val) === '[object Array]'
```

Comment:
`Array.isArray`?

Reply:
I just moved this function, didn't try and make any changes (I agree though you're right)